### ISSUE TITLE
[Suggestion] Strictly Better AH Bot

### ISSUE THREAD

**Issue by StraysFromPath** [ZhengPeiRu21/mod-individual-progression#205]
The prebuild uses this AH bot:

https://github.com/azerothcore/mod-ah-bot/issues/78

This AH bot has lots of weird issues such as

1) no enchanting mats
2) large numbers of banned/disabled items which real players put on the AH
3) item values are a bad form of random, meaning rare expense wotlk profession mats can go for 50s/1g but lvl 10 greens can go for 10g 

all of which are fixed by this fork which uses real human auction house data to assign prices/stacks/ah items:

The fork is the same bot but massively improved/fixed:

https://github.com/DedmenWoW/mod-ah-bot

**Comment by ZhengPeiRu21** [comment]
Thanks for the suggestion, I'll switch out to this module in the next update.

**Comment by Grimfeather** [comment]
https://i.gyazo.com/e7b94c54b3db3bb7d725b9e567cfc01d.png

This forked module has no issues section, and google didn't have the answer either so I hope it's ok for me to ask this here.
I tried today to include this ah bot but I'm running into this problem where it can't find the acore_world mod_auctionhousebot_priceoverride table when starting the server. This table is indeed missing.
https://i.gyazo.com/20ff530465c47ed674d97e18bad1f8d6.png

I did run the sql that was included, but the sql doesn't add, modify or even mention the priceoverride table.

What did I do wrong / what did I miss?
